api/institutions/views.py

```diff
@@ -366,3 +377,64 @@ def create(self, *args, **kwargs):
         except RelationshipPostMakesNoChanges:
             return Response(status=status.HTTP_204_NO_CONTENT)
         return ret
+
+
+class InstitutionDepartmentList(JSONAPIBaseView, ListFilterMixin, generics.ListAPIView, InstitutionMixin):
+    permission_classes = (
+        drf_permissions.IsAuthenticatedOrReadOnly,
+        base_permissions.TokenHasScope,
+        IsInstitutionalMetricsUser,
+    )
+
+    serializer_class = InstitutionDepartmentSerializer
+
+    required_read_scopes = [CoreScopes.INSTITUTION_READ_METRICS]
+    required_write_scopes = [CoreScopes.NULL]
+
+    view_category = 'institutions'
+    view_name = 'institution-department-metrics'
+
+    ordering = ('-number_of_users', 'name',)
+
+    @classmethod
+    def _get_departments_counts(cls, current_user_data: list) -> dict:
```

Comment:
I don't believe summing the project counts of users in a given department will work. There may be multiple users in the same department working on the same project and I believe that should count as 1 project for the department.

Reply:
er- actually, perhaps not. Depends on if the count for a department is the number of distinct projects associated with a given department (through users) or the total number of projects for users associated with a given department.

Reply:
Unless I got the recs wrong, this just counts the number of users in each department, no project counts involved. Return json should look like [this](https://github.com/CenterForOpenScience/osf.io/pull/9318/files#diff-11dee4e28adbf356065341776ce922b7R110-R133)

Reply:
Ah looks like you're right, sorry about that.